source/Octo.Tests/Commands/ReleasePlanBuilderTests.cs

```diff
@@ -28,7 +28,7 @@ public class ReleasePlanBuilderTests
         IChannelVersionRuleTester versionRuleTester;
         IOctopusAsyncRepository repository;
         IDeploymentProcessRepository deploymentProcessRepository;
-        IDeploymentProcessRepositoryBeta deploymentProcessRepositoryBeta;
+        IDeploymentProcessBetaRepository deploymentProcessRepositoryBeta;
```

Comment:
this made me curious, so I went looking and; yep! our naming is inconsistent over in Client. I've [raised a PR](https://github.com/OctopusDeploy/OctopusClients/pull/609) to bring it into line.

Reply:
noice